src/syntaxtree.c

```diff
@@ -210,7 +322,6 @@ static Obj SyntaxTreeRecExpr(Obj result, Expr expr)
 
     len = SIZE_EXPR(expr) / (2 * sizeof(Expr));
     list = NEW_PLIST(T_PLIST, len);
-    SET_LEN_PLIST(list, len);
```

Comment:
Hm, did you already rebase on the latest `mh/syntaxtree`?

Reply:
Yes, but this was still there . . .

Reply:
In the version I just merged into master, they definitely are not there anymore...